### PR TITLE
Fix list highlight on Customer Review Select #2646

### LINKED ISSUE #2646: Fix list highlight on Customer Review Select
**Actual Behaviour**

On selecting customer review while browsing skills, blue outlined div comes up

**Expected Behaviour**

Fix it

**Steps to reproduce it**

susi.ai ---> Try to filter by customer review

**Screenshots of the issue**

Screenshot 2019-07-21 at 4 03 26 PM

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2646, #2645, #2648, #2651

Changes: 
- [x] Fix list highlight on Customer Review Select
- [x] Added list highlight on hover
- [x] Scroll to top on metric change
- [x] Fix settings height overflow
- [x] Fix browseskill add url params on filter apply(Half issue)
- [x] Fixed avatar circular image

Demo Link: https://pr-2647-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Screenshot 2019-07-21 at 5 02 48 PM

Fixed this:

Screenshot 2019-07-21 at 5 08 19 PM

Screenshot 2019-07-21 at 8 48 08 PM

Screenshot 2019-07-21 at 8 48 41 PM

### CODE REVIEW

File: src/components/cms/BrowseSkill/BrowseSkill.js [hunk 1]
```diff
@@ -410,12 +409,14 @@ class BrowseSkill extends React.Component {
...
     }
     this.props.actions.getSkills(payload);
+    scrollAnimation(document.documentElement, 0, 200, 'vertical');
   };
 
   loadMetricsSkills = () => {
     this.props.actions.getMetricsSkills({
       languageValue: this.props.languageValue,
     });
+    scrollAnimation(document.documentElement, 0, 200, 'vertical');
```
Comment (akshatnitd) [inline src/components/cms/BrowseSkill/BrowseSkill.js]: How about implementing in the `withTracker` HOC in your other PR?

Comment (fragm3) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: Oh, nice idea. But we would require inline scroll calls as well for metrics change on browseskill

Comment (akshatnitd) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: To handle that, we should actually add the url param on metric change to the url as well.

**Comment by fragm3** [comment]
@akshatnitd done, https://github.com/fossasia/susi.ai/pull/2634#discussion_r305622193

File: src/components/cms/BrowseSkill/BrowseSkill.js (hunk 1 above)
Comment (fragm3) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: Okay working on it, created issue https://github.com/fossasia/susi.ai/issues/2651

**Comment by fragm3** [comment]
@akshatnitd I have integrated query string in browse skill. I will pick up order maintain in the next PR.

**Note**: I had implemented queryString for **pages, entries per page** as well, but didn't won't work well as we splitting the skills on frontend instead of backend. So for now, removed that code.

**Comment by fragm3** [comment]
@akshatnitd fixed the link

**Pull request merged**